src/utils/parser/constants.mjs

```diff
@@ -71,62 +71,32 @@ export const DOC_API_HEADING_TYPES = [
 
 // This is a mapping for types within the Markdown content and their respective
 // JavaScript primitive types within the MDN JavaScript docs
-// @see DOC_MDN_BASE_URL_JS_PRIMITIVES
-export const DOC_TYPES_MAPPING_PRIMITIVES = {
-  boolean: 'Boolean',
-  integer: 'Number', // Not a primitive, used for clarification.
-  null: 'Null',
-  number: 'Number',
-  string: 'String',
-  symbol: 'Symbol',
-  undefined: 'Undefined',
-};
+// @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Data_structures#primitive_values
+export const DOC_TYPES_MAPPING_PRIMITIVES = [
+  'null',
+  'undefined',
+  'boolean',
+  'number',
+  'bigint',
+  'string',
+  'symbol',
+];
 
 // This is a mapping for types within the Markdown content and their respective
 // JavaScript globals types within the MDN JavaScript docs
 // @see DOC_MDN_BASE_URL_JS_GLOBALS
 export const DOC_TYPES_MAPPING_GLOBALS = {
   ...Object.fromEntries(
     [
-      'AggregateError',
-      'Array',
-      'ArrayBuffer',
-      'DataView',
-      'Date',
-      'Error',
-      'EvalError',
-      'Function',
-      'Map',
-      'NaN',
-      'Object',
-      'Promise',
-      'Proxy',
-      'RangeError',
-      'ReferenceError',
-      'RegExp',
-      'Set',
-      'SharedArrayBuffer',
-      'SyntaxError',
-      'Symbol',
-      'TypeError',
-      'URIError',
-      'WeakMap',
-      'WeakSet',
-
-      'TypedArray',
-      'Float16Array',
-      'Float32Array',
-      'Float64Array',
-      'Int8Array',
-      'Int16Array',
-      'Int32Array',
-      'Uint8Array',
-      'Uint8ClampedArray',
-      'Uint16Array',
-      'Uint32Array',
+      // This is updated with every ES-spec, so, as long as the
+      // `globals` package is up-to-date, so will our globals
+      // list.
+      ...Object.keys(globals.builtin),
```

Comment:
Shouldn't we specify to be Node's globals? Or does this .builtin checks what globals are available?

Reply:
No, this is only checking V8's globals, since these are all going to link to MDN. The Node.js-specific globals are their own object, defined elsewhere in this file.